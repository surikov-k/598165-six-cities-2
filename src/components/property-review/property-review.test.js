import React from 'react';
import renderer from 'react-test-renderer';

import PropertyReview from './property-review.jsx';

const review = {
  rating: 0,
  comment: ``,
  date: 0,
  userName: ``,
  userAvatar: ``,
};

describe(`<PropertyReview/>`, () => {
  it(`renders correctly`, () => {
    const component = renderer
      .create(<PropertyReview
        review={review}
      />)
      .toJSON();

    expect(component).toMatchSnapshot();
  });
});

