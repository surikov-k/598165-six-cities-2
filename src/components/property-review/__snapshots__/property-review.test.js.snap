// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PropertyReview/> renders correctly 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src=""
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "0%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      
    </p>
    <time
      className="reviews__time"
      dateTime={1970-01-01T00:00:00.000Z}
    >
      January 1970
    </time>
  </div>
</li>
`;
